src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -453,7 +453,7 @@ public static <T> T doAs(final Subject subject,
     }
 
     /**
-     * Perform privileged work as a particular {@code Subject}.
+     * Perform work as a particular {@code Subject}.
```

Comment:
The new callAs is specified as "Executes a Callable with subject as the current subject".  ScopedValue uses "Runs an operation with each scoped value in this mapping bound to its value in the current thread" and "Calls a value-returning operation with each scoped value in this mapping bound to its value in the current thread".

Looking at the doAs and doAsPrivileged methods makes me wonder if "Perform work" should be replaced rather than just dropping the word "privileged". That is, maybe these methods should are re-worded to specify that they run an action rather than "perform work". I realise this is more than what this PR is about, I'm just observing that "Perform work" is strange to see in the API docs.

Reply:
Well the "perform work" words have been there for a long time. Perhaps "Performs the specified action as a particular subject" which would be more consistent with `AccessController.doPrivileged`. But, I'm also wary of making too many changes.